index.html

```diff
@@ -14,9 +14,40 @@
     href="styles/materialize/css/materialize.min.css"
     media="screen,projection"
   />
+  <link
+    href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    rel="stylesheet"
+  />
+
   <!-- personal stylesheet-->
   <link rel="stylesheet" type="text/css" href="styles/style.css" />
 </head>
+
 <body>
+  <!-- Nav Bar -->
+  <nav id="nav-bar" class="grey">
+    <div class="nav-wrapper container">
+      <a href="#" data-target="mobile-nav-links" class="sidenav-trigger"
+        ><i class="material-icons">menu</i></a
+      >
+      <a href="/index.html" id="logo-container" class="brand-logo"
+        >Assignment 1</a
+      >
+      <!-- Nav Links -->
+      <ul id="nav-links" class="right hide-on-med-and-down">
+        <li><a href="quymbee.html">Quymbee</a></li>
+        <li><a href="david.html">David</a></li>
+      </ul>
+      <!-- #nav-links end -->
+    </div>
+  </nav>
+  <!-- #nav-bar end -->
+
+  <!-- Mobile Nav Links -->
```

Comment:
niceee on the nav bar 👏 ! made one smol change to make it more mobile friendly.
followed [materialize extended navbar](https://materializecss.com/navbar.html#navbar-tabs) example